cookbooks/hdfsdu/templates/default/hdfsdu.upstart.conf.erb

```diff
@@ -0,0 +1,9 @@
+#!upstart
+description "HDFS Disk Usage"
+
+chdir <%= "#{node[:hdfsdu][:install_dir]}/hdfsdu" %> 
+exec sudo -u <%= "#{node[:hdfsdu][:user]}" %> java -Xmx<%= node[:hdfsdu][:max_memory] %> -cp 'lib/*' \
```

Comment:
Can (and should) you specify the user in the Upstart job rather than use `sudo -u` here?